### PR TITLE
change vs download link

### PR DESCRIPTION
**Pull request by tdykstra**
Per Scott's suggestion, this might be a better link for the VS install.

https://www.visualstudio.com/downloads/

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  I agree that's simpler but currently it has preview links that go to Download Visual Studio 2017 Preview version 15.3 
I'm guessing they'll clean that up. Maybe they'll leave it until the come out with 15.4 preview in a couple months.

File: aspnetcore/includes/install2.0.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Install the following:
 
 * [.NET Core 2.0.0 SDK](https://dot.net/core) or later.
-* [Visual Studio](https://www.visualstudio.com/vs/) 15.3 or later.
\ No newline at end of file
+* [Visual Studio](https://www.visualstudio.com/downloads/) 15.3 or later, with the **ASP.NET and web development** and **.NET Core cross-platform development workloads** installed.
```
Comment (Rick-Anderson) [inline aspnetcore/includes/install2.0.md]: can we break that up into 2 sentences?  maybe
[Visual Studio](https://www.visualstudio.com/downloads/) 15.3 or later, with the the following workloads:

* first
* second

Comment (Rick-Anderson) [inline aspnetcore/includes/install2.0.md]: Do we need/want a screenshot? Dan.Roth doesn't like screen shots. He was under the impression we could remove all of that with one install link - for .NET Core 2

**Comment by Rick-Anderson** [comment]
Last week Dan said just send them to https://dot.net/core 
When I tested that link it had a console tab and VS17 tab

**Comment by tdykstra** [comment]
I wouldn't add screenshots, but we could link to https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio for info about workload installs

**Pull request merged**